hoomd/HOOMDMath.h

```diff
@@ -349,6 +349,50 @@ inline HOSTDEVICE double pow(double x, double y)
     return ::exp(y * log(x));
     }
 
+//! Compute the pow of x,y in double precision when y is an integer using squaring
+inline HOSTDEVICE double pow(double x, unsigned int y){
```

Comment:
Add a `hoomd/test/test_math.cc` file using UPP11 to unit test these new math functions.